references/detection/train.py

```diff
@@ -41,8 +47,15 @@ def get_dataset(name, image_set, transform, data_path):
     return ds, num_classes
 
 
-def get_transform(train, data_augmentation):
-    return presets.DetectionPresetTrain(data_augmentation) if train else presets.DetectionPresetEval()
+def get_transform(train, args):
+    if train:
+        return presets.DetectionPresetTrain(args.data_augmentation)
+    elif not args.weights:
+        return presets.DetectionPresetEval()
+    else:
+        fn = PM.detection.__dict__[args.model]
```

Comment:
should we also check for PM here and raise exception?

Reply:
Yeah we could/should. 

Note that this is temporary for as long as we check the models, so I don't want to have a massive amount of error handling spread all over the place. Perhaps a middle ground is to move the existing exception somewhere where it will be triggered if the PM is None once. I would prefer to follow up on a different PR because this change needs to happen across all scripts. Thoughts?

Reply:
I am ok with that. approving the PR in that case

Reply:
Thanks I'll send a new PR in a bit.